node/core/av-store/Cargo.toml

```
@@ -31,3 +32,4 @@ kvdb-memorydb = "0.8.0"
 sp-core = { git = "https://github.com/paritytech/substrate", branch = "master" }
 polkadot-node-subsystem-util = { path = "../../subsystem-util" }
 polkadot-node-subsystem-test-helpers = { path = "../../subsystem-test-helpers" }
+sp-keyring = { git = "https://github.com/paritytech/substrate", branch = "master" }
```

trailing new line is missing

```suggestion
sp-keyring = { git = "https://github.com/paritytech/substrate", branch = "master" }

```

---

I have the editorconfig plugin for VSCode and the `.editorconfig` has `insert_final_newline=true`. I don't know what else I could do to get it to save with trailing newline

---

VSCode has built-in `Insert Final Newline` setting. I don't know why it's not picked up from the editorconfig though.

---

Probably because it doesn't specify it for toml files
https://github.com/paritytech/polkadot/blob/master/.editorconfig